backend/server/routers/user.py

```diff
@@ -145,5 +145,8 @@ def update_degree_length(numYears: int, token: str = DUMMY_TOKEN):
     if diff > 0:
         user['planner']['years'] += ([{"T0": [], "T1": [], "T2": [], "T3": []}] * diff)
     else:
+        for year in user['planner']['years'][diff:]:
+            for term in year.values():
+                user['planner']['unplanned'] += term
```

Comment:
this is super nitpicky but, a `user[..][..].extend(term)` makes the collection-ness of `unplanned` alot easier to see

otherwise, ```
user['planner']['unplanned'].extend(sum(year.values()))
```

pick your poison - (also i cannot stress enough how much this doesn't matter lol)

Reply:
I would've prefer the latter, but it gave me errors so I went with the first suggestion thanks.